exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = true; // finish this statement
```

Comment:
Finish each statement using comparison operators and the variables provided to generate a boolean value. For example compare the variables ```studentCount``` and ```mentorCount``` to check if there are more mentors than students. If so the statement will evaluate to true.